docs/static/include/pluginbody.asciidoc

```diff
@@ -697,6 +697,57 @@ For more examples of output plugins, see the https://github.com/logstash-plugins
 
 endif::receive_method[]
 
+ifdef::event_factory_method[]
+
+===== `event_factory` Method
```

Comment:
I would suggest an anchor here, but because this content is resused, it would not be unique and would blow things up. 💥  I believe that we could work around by using the `id="plugins-{type}s-{plugin}-` construct, but that's not our highest priority right now.